ui/js/main.js

```diff
@@ -2,13 +2,19 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import lbry from './lbry.js';
 import lbryio from './lbryio.js';
+
+const i18n = require('y18n')({directory: 'app/locales'/*, locale: lbry.getClientSetting('language')*/});
+window.__ = i18n.__;
+window.__n = i18n.__n;
```

Comment:
Yes, the translate functions are used on every page. __n is used rarely and we could use it as i18.__n() instead, though to make it equivalent and available.
I tried it first with i18n component in local scope but problems start with language switching. It wouldn't change the language strings if on the settings page we load the i18n again and do a setLanguage(language) method call. We could tell the user to restart the app, which would have been much easier for me, but I didn't like the idea of app restart if you changed the language. More than open for other suggestions ?